Library/Homebrew/cmd/update.sh

```diff
@@ -236,7 +237,8 @@ homebrew-update() {
       --help) brew help update; exit $? ;;
       --verbose) HOMEBREW_VERBOSE=1 ;;
       --debug) HOMEBREW_DEBUG=1;;
-      --rebase) HOMEBREW_REBASE=1 ;;
+      --merge) HOMEBREW_MERGE=1 ;;
+      --rebase) HOMEBREW_REBASE=1 ;; # a no-op now.
```

Comment:
Let's not populate a global variable (that is no longer used) for a no-op. We could also omit this line completely, as `--rebase` will be handled (ignored) further down by `--*) ;;`.